### PR TITLE
#103 Add 'www' subdirectory to Docs, Add Jenkinsfile for Public Docs build

### PR DESCRIPTION
**Pull request by bsmith-n4**
__Issue:__
https://github.com/eclipse/n4js/issues/103

- [x] Build

__After Merging:__ 
Jenkins Build config must point to eclipse/n4js for Jenkinsfile location (currently points to my fork)

Signed-off-by: bsmith <brian.smith@numberfour.eu>

### CODE REVIEW

File: docs/www/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env groovy
+/*
+ * Copyright (c) 2016 NumberFour AG.
```
Comment (torstenkraemer) [inline docs/www/Jenkinsfile]: This header seems a little bit behind...

File: docs/www/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+
+// email configuration
+def mailSender = 'robot@numberfour.eu'
+def String recipient = 'brian.smith@numberfour.eu'
+def String replyTo = 'brian.smith@numberfour.eu'
+def String team = 'brian.smith@numberfour.eu, staff-devtools@numberfour.eu'
+def String buildhtml = env.BUILD_URL+"execution/node/5/ws/sourceX/docs/www/web-html"
```
Comment (torstenkraemer) [inline docs/www/Jenkinsfile]: The mail config should be de-personalized. Cf. mail config in https://github.com/eclipse/n4js/blob/master/Jenkinsfile

File: docs/www/Jenkinsfile (hunk 1 above)
Comment (bsmith-n4) [inline docs/www/Jenkinsfile]: :) updated

File: docs/www/Jenkinsfile (hunk 2 above)
Comment (bsmith-n4) [inline docs/www/Jenkinsfile]: Done

**Comment by bsmith-n4** [comment]
Problems with commit signing by GitHub merge - passing PR open here: https://github.com/eclipse/n4js/pull/112

**Pull request closed without merging**